Assets/Scripts/MoveStreet.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public sealed class MoveStreet : MonoBehaviour {
+
+	[SerializeField]
+	private float speedStreet = -1;
+	const float MAX_DISTANCE = -100;
+
+	private void Update () {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'Update'.](https://www.codacy.com/app/mendes-barreto/BSToysGame/pullRequest?prid=684733)